style.css

```diff
@@ -0,0 +1,4 @@
+.body {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
- [ ] When using Bootstrap there is no need to reset the browser's default stylesheet as bootstrap already normalizes or resets these values.